RPerformanceTracking/Private/_RPTSender.m

```diff
@@ -220,11 +211,11 @@ - (void)writeMeasurement:(_RPTMeasurement *)measurement metricId:(NSString *)met
     _sentCount++;
 }
 
-- (NSInteger)indexAfterSendingWithStartIndex:(NSInteger)startIndex endIndex:(NSInteger)endIndex
+- (NSUInteger)indexAfterSendingWithStartIndex:(NSUInteger)startIndex endIndex:(NSUInteger)endIndex
 {
-    NSInteger returnIndex = startIndex;
-    // if _sentCount <= 0, don't send metric.
-    if (_sentCount <= 0)
+    NSUInteger returnIndex = startIndex;
+    // if _sentCount == 0, don't send metric.
+    if (_sentCount == 0)
```

Comment:
I think these changes to unsigned int are probably ok.

The code was originally ported from [Java](https://github.com/rakutentech/android-perftracking/blob/master/Core/src/main/java/com/rakuten/tech/mobile/perf/core/Sender.java#L39) and there it's just `int`so that's probably where the logic here came from. I didn't realise Java had this [issue](https://stackoverflow.com/a/25556323) and int can't be unsigned.